src/patternfly/components/Badge/examples/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Documentation from '@siteComponents/Documentation';
+import Example from '@siteComponents/Example';
+import docs from '../docs/code.md';
+import ReadBadge from './read-badge.hbs';
```

Comment:
We also made up some guidelines for consistently naming the example files and variables names in the JS. For the moment, that's all in the contribution doc.

Reply:
Ah! Thanks, I'll consult the contribution doc. I followed the var names in the Alert component here.